lambdas/postgres-migration-async-operation/main.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  lambda_path      = "${path.module}/dist/webpack/lambda.zip"
+  all_bucket_names = [for k, v in var.buckets : v.name]
+}
+resource "aws_lambda_function" "postgres-migration-async-operation" {
+  function_name    = "${var.prefix}-postgres-migration-async-operation"
+  role             = aws_iam_role.postgres_migration_async_operation_role.arn
+  filename         = local.lambda_path
+  source_code_hash = filebase64sha256(local.lambda_path)
+  handler          = "index.handler"
+  runtime          = "nodejs12.x"
+  timeout          = 300
+  memory_size      = 512
+
+  environment {
+    variables = {
+      databaseCredentialSecretArn  = var.rds_user_access_secret_arn
+      dbHeartBeat                  = var.rds_connection_heartbeat
+      system_bucket                = var.system_bucket
+      AsyncOperationsTable         = var.dynamo_tables.async_operations.name
+      stackName                    = var.prefix
+      MigrationLambda              = var.data_migration2_function_arn
+      EcsCluster                   = var.ecs_cluster_name
+      AsyncOperationTaskDefinition = var.async_operation_task_definition_arn
```

Comment:
Nit: consider making this consistent with the signature of `startAsyncOperation` so you can assign process.env.startAsyncOperation directly to a variable, then not re-assign it in the function invocation.

Reply:
Super Nit: alpha-sort